spec/features/contribute/tish_library_reward_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+require 'ffaker'
+include Warden::Test::Helpers
+
+RSpec.feature 'submit a Tish Library Reward contribution' do
+  let(:user) { FactoryGirl.create(:user) }
+  let(:csv_path) { Rails.root.join('config', 'deposit_type_seed.csv').to_s }
+  let(:importer) { DepositTypeImporter.new(csv_path) }
+  let(:pdf_path) { Rails.root.join('spec', 'fixtures', 'hello.pdf') }
+  before do
+    login_as user
+    importer.import_from_csv
```

Comment:
@little9 does this need to happen before each test, or just once? I'm wondering whether it should go in `before(:suite)` in `spec_helper` or `rails_helper`